src/plans/create/CreatePlanWizard.tsx

```diff
@@ -1,15 +1,13 @@
-import React, { FC, useState } from 'react';
+import { FC, useState } from 'react';
 import { FormProvider, useForm } from 'react-hook-form';
 
 import { Form, Title, Wizard, WizardStep, WizardStepType } from '@patternfly/react-core';
 import { isEmpty } from '@utils/helpers';
 import { useForkliftTranslation } from '@utils/i18n';
 
-import { GeneralInformationForm } from './steps/GeneralInformationForm';
 import { firstStep, planStepNames, planStepOrder, PlanWizardStepId } from './constants';
 import { CreatePlanWizardFooter } from './CreatePlanWizardFooter';
-
-import './CreatePlanWizard.style.scss';
+import GeneralInformationStep from './steps/general-information/GeneralInformationStep';
 
 export const CreatePlanWizard: FC = () => {
   const { t } = useForkliftTranslation();
```

Comment:
the comment aobut typing was refering to useForm
`useForm<SomeFormType>`

Reply:
Also i see in line 16,17 watch, better to use useWatch , it lets rendering

Reply:
The watch is only there to print formValues on the review step temporarily, so that doesn't matter right now.

Reply:
The first type that useForm takes is `FieldValues`, which is what I mentioned in slack. The key/values can be anything right now. I can create a type, but it wouldn't be complete, since I only know for certain types that are on this General information step, is that what you'd like? Otherwise, I'm not sure what the ask is for the form's type.

Reply:
I've created a type to capture what field value types we should expect and some supportive hooks that are typed around these values as well, so we can rely more on type inference within components.